src/main/java/org/cristalise/kernel/lookup/ItemPath.java

```diff
@@ -178,10 +191,21 @@ public String getName() {
     public String getClusterPath() {
         return ClusterType.PATH + "/Item";
     }
-    
+
     public static boolean isUUID(String entityKey) {
         if (entityKey.startsWith("/entity/")) entityKey = entityKey.substring(8);
 
         return entityKey.matches("^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$");
     }
+
+    /**
+     * Some item paths should have no '/entity' in the path. e.g workflow '/desc/ActivityDesc/aaa/ProductWorkflow'
+     * or '/aaa/SiteFactory'
+     * @return
+     */
+    @Override
+    public String getStringPath() {

```

Comment:
This change does not seem to me correct. See my comment on `dependencyMemberPath`